diff --git a/releng/devkit.py b/releng/devkit.py
index 139051a..2e4a285 100755
--- a/releng/devkit.py
+++ b/releng/devkit.py
@@ -511,7 +511,7 @@ def compute_umbrella_header_path(frida_root, host, flavor, package, umbrella_hea
         return os.path.join(frida_root, "build", "frida" + flavor + "-" + host, *prefix, "include", *umbrella_header)
 
 def sdk_lib_path(name, frida_root, host):
-    return os.path.join(frida_root, "build", "sdk-windows", msvs_arch_config(host), "lib", name)
+    return os.path.join(frida_root, "build", "sdk-windows", msvs_arch_config(host), "lib-dynamic", name)
 
 def internal_include_path(name, frida_root, host):
     return os.path.join(frida_root, "build", "tmp-windows", msvs_arch_config(host), name + msvs_arch_suffix(host))
diff --git a/releng/frida-Debug.props b/releng/frida-Debug.props
index 8731794..d1f32d3 100644
--- a/releng/frida-Debug.props
+++ b/releng/frida-Debug.props
@@ -12,7 +12,7 @@
     <ClCompile>
       <Optimization>Disabled</Optimization>
       <BasicRuntimeChecks>EnableFastChecks</BasicRuntimeChecks>
-      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
 
       <PreprocessorDefinitions>_DEBUG;_HAS_ITERATOR_DEBUGGING=0;%(PreprocessorDefinitions)</PreprocessorDefinitions>
diff --git a/releng/frida-Release.props b/releng/frida-Release.props
index 46cc0d7..d11877f 100644
--- a/releng/frida-Release.props
+++ b/releng/frida-Release.props
@@ -13,7 +13,7 @@
       <FavorSizeOrSpeed>Size</FavorSizeOrSpeed>
       <WholeProgramOptimization>false</WholeProgramOptimization>
       <StringPooling>true</StringPooling>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <BufferSecurityCheck>false</BufferSecurityCheck>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <FloatingPointModel>Fast</FloatingPointModel>
diff --git a/releng/frida.props b/releng/frida.props
index bc4fc03..83fb37f 100644
--- a/releng/frida.props
+++ b/releng/frida.props
@@ -35,7 +35,7 @@
     </ClCompile>
     <Link>
       <ImportLibrary>$(IntDir)$(ProjectName).lib</ImportLibrary>
-      <AdditionalLibraryDirectories>$(IntDir)..\gum-32;$(IntDir)..\gum-64;$(IntDir)..\gumjs-32;$(IntDir)..\gumjs-64;$(SolutionDir)build\sdk-windows\$(PlatformName)-$(ConfigurationName)\lib;$(SolutionDir)build\sdk-windows\$(PlatformName)-$(ConfigurationName)\lib\gio\modules;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(IntDir)..\gum-32;$(IntDir)..\gum-64;$(IntDir)..\gumjs-32;$(IntDir)..\gumjs-64;$(SolutionDir)build\sdk-windows\$(PlatformName)-$(ConfigurationName)\lib-dynamic;$(SolutionDir)build\sdk-windows\$(PlatformName)-$(ConfigurationName)\lib-dynamic\gio\modules;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <AdditionalDependencies>libffi.a;libgee-0.8.a;libgio-2.0.a;libgioopenssl.a;libglib-2.0.a;libgmodule-2.0.a;libgobject-2.0.a;libgthread-2.0.a;libjson-glib-1.0.a;libminizip.a;libnice.a;libsoup-2.4.a;libsqlite3.a;libusrsctp.a;libcapstone.a;libquickjs.a;libtcc.a;libv8-8.0.a;libcrypto.a;libssl.a;libz.a;libbrotlicommon.a;libbrotlienc.a;libbrotlidec.a;crypt32.lib;dnsapi.lib;iphlpapi.lib;ole32.lib;psapi.lib;secur32.lib;shlwapi.lib;winmm.lib;ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
   </ItemDefinitionGroup>
